packages/react-composites/src/composites/CallComposite/hooks/useHandlers.ts

```diff
@@ -229,6 +229,54 @@ const createCompositeHandlers = memoizeOne(
       onMuteAllRemoteParticipants: async (): Promise<void> => {
         await adapter.muteAllRemoteParticipants();
       },
+      /* @conditional-compile-remove(media-access) */
+      onForbidAudio: capabilities?.forbidOthersAudio?.isPresent
+        ? async (userIds: string[]): Promise<void> => {
```

Comment:
Can we double check here that we have to use strings? for the composites the CallingSDK is a dependency so we can use the typing there here if it would make more sense to be more type safe

Reply:
![image](https://github.com/user-attachments/assets/13404fb8-41b1-4294-ad51-882ca9883975)
Good idea but similar to other functions, if we want to change it need to change them all. 
Let's follow the existing pattern for now.